docs/content/installation/installation-with-manifests.md

```diff
@@ -51,13 +51,10 @@ This document describes how to install the NGINX Ingress Controller in your Kube
 
 ## 2. Create Common Resources
 
-In this section, we create resources common for most of the Ingress Controller installations:
-1. Create a secret with a TLS certificate and a key for the default server in NGINX:
-    ```
-    $ kubectl apply -f common/default-server-secret.yaml
-    ```
+**Note**: If you want to use your own certificate and key for TLS tor the NGINX Ingress controller `default_server`, you are free to do so. The format should be: `namespace/name_of_secret`   
```

Comment:
If you want to use the callout formatting, use 

```
{{< note >}} 
You can use your own certificate and key for TLS for the NGINX Ingress Controller `default_server`. Define the  secret and the namespace where it resides using the format: `namespace/name_of_secret`.
{{< /note >}}
```

Reply:
Made the changes that were recommended.